brainstormforce/ultimate-addons-for-gutenberg
# UAG-436 Fix: Table of contents - Not preview on page when Buddy Boss Platform is activated.

### Description

- https://ua-gutenberg.atlassian.net/browse/UAG-436

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install BuddyBoss Platform.
- Drag and drop UAG TOC block on-page.
- It will not preview in the editor throw a console error.
- Switch to `toc-conflicting-with-buddypress-plugin` and refresh the page and check again it will render on the page.

### Checklist:
